File: 준영/프로그래머스/LV2/올바른 괄호.py
def solution(s):
    answer = True
    stack = []
    for i in s:
        if i == '(':
            stack.append('(')
        else:
            if len(stack) > 0:
                stack.pop()
            else:
                answer = False
                break
    if len(stack) > 0:
        answer = False
    return answer

File: 준영/프로그래머스/LV2/올바른 괄호.js
function solution(s){
  var answer = true
  const stack = []
  for (const i of s) {
    if (i == '(') {
      stack.push('(')
    } else {
      if (stack.length > 0) {
        stack.pop()
      } else {
        answer = false
        break
      }
    }
  }
  if (stack.length > 0) {
    answer = false
  }
  return answer
}